### ISSUE TITLE
Cannot Unpublish/Archive/Delete a Pathfinder page

### ISSUE THREAD

**Issue by jakxnz** [codecraft/silverstripe-pathfinder#7]
When I try to delete a Pathfinder page from the site tree, I cannot see the option to do so

## Steps to reproduce

1. Login to the CMS and navigate to a Pathfinder page
2. Attempt to unpublish, archive or delete it

**Expected result**

I can unpublish, archive or delete the page

**Actual result**

I get hit with an error:

```
Uncaught LogicException: DataObject::delete() called on a DataObject without an ID
```

**Comment by jakxnz** [comment]
Fixed in 0.1

**Comment by jakxnz** [comment]
Fixed in 0.1
